_pages/mobile-interview-project.md

```diff
@@ -73,14 +88,54 @@ In the case that no employees are returned, the application should present an em
         ...
       ]
     }
-~~~
+```
+
+# Other Details
+
+## Use What You Know
+
+We’re looking to see you at your best – as such, please use the language and frameworks you’re most comfortable with on your chosen mobile platform. This means that if you’re most proficient with Objective-C or Java, use that – don’t feel pressured to use Swift or Kotlin!
+
+Similarly, don’t feel pressured to use SwiftUI, Compose, and/or other libraries and techniques that you are unfamiliar with. We’re not going to prefer answers in one language over another, but we will expect you to be comfortable and proficient with your choices.
+
+## Show Us Your Strengths
+
+As you’re working on your app, feel free to focus your time and energy on your areas of expertise. If you’re a really great architect, focus on that. If you’re really great at building UI, focus on that. We really want to see what you’re best at.
```

Comment:
I like the idea here. I'm a little worried it will encourage folks who are great at UI to leave crummy architecture, and then we _will_ penalize them for that. There's a certain level of architecture soundness we do require, yes? Same for UI, although less so. But I've certainly marked folks down a bit if all their images are different sizes, say.

Reply:
Yup the next header is supposed to address that we definitely do want to see their best code in addition to their strengths